src/components/views/article/Article.js

```diff
@@ -0,0 +1,122 @@
+
+import React, { Fragment, useEffect } from 'react';
+import ReactHtmlParser from 'react-html-parser';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getArticleBySlug } from '@Redux/actions/articleActions';
+import Tag from '@Common/tags/Tag';
+import ArticleComment from '@Common/comments/ArticleComment';
+import parseDataFromJSON from '@Utils/parseEditorData';
+import './article.scss';
+
+const Article = ({
+  getArticleBySlug: fetchSingleArticleBySlug,
+  match,
+  singleArticle,
+  loading,
+  error,
+}) => {
+  useEffect(() => {
+    const getArticle = async () => {
+      await fetchSingleArticleBySlug(match.params.slug);
+    };
+
+    getArticle();
+  }, [match, fetchSingleArticleBySlug]);
+
+  return (
+    <Fragment>
+      <div className="article__container">
+        {loading
+          && (
+          <center>
+            <h1>Loading...</h1>
```

Comment:
We have a loader/spinner, won't it be suitable here?

Reply:
Yes, I'll add that